pkgs/tools/audio/beets/common.nix

```
@@ -69,6 +69,7 @@ python3Packages.buildPythonApplication rec {
 
   # see: https://github.com/NixOS/nixpkgs/issues/56943#issuecomment-1131643663
   nativeBuildInputs = [
+    python3Packages.sphinxHook
```

```suggestion
    sphinxHook
```

Import the top-level attribute instead.